packet.go

```diff
@@ -302,3 +314,230 @@ func parsePCR(i *astikit.BytesIterator) (cr *ClockReference, err error) {
 	cr = newClockReference(int64(pcr>>15), int64(pcr&0x1ff))
 	return
 }
+
+func writePacket(w *astikit.BitsWriter, p *Packet, targetPacketSize int) (written int, retErr error) {
```

Comment:
I'd be inclined to keep this method but also export in `muxer.go` a `func (m *Muxer) WritePacket(p *Packet) (written int, retErr error)` that would simply call `writePacket`

Reply:
Done